### PR TITLE
Procedures support update

### PR DESCRIPTION
**Pull request by sjpadgett**
styled code pickers and updated dependencies.
minor style of procedure configuration and dependency update.
consolidated procedure form to better handle questions and slim down form height.

### CODE REVIEW

File: interface/orders/types.php [hunk 1]
```diff
@@ -63,230 +66,241 @@
...
-  <th class='col1' align='left'>&nbsp;&nbsp;<?php xl('Name', 'e') ?></th>
-  <th class='col2' align='left'><?php xl('Order', 'e') ?></th>
-  <th class='col3' align='left'><?php xl('Code', 'e') ?></th>
-  <th class='col4' align='left'><?php xl('Description', 'e') ?></th>
-  <th class='col5' align='left'>&nbsp;</th>
- </tr>
-</table>
-
-<div id="con0">
+    <h3 style='margin-top:0'><?php xlt('Types of Orders and Results') ?></h3>
```
Comment (bradymiller) [inline interface/orders/types.php]: need an echo here, or else it will be blank

**Comment by bradymiller** [comment]
I only found one issue on code review; otherwise code looks good to go. If testing well for you, then feel free to bring into codebase.

File: interface/orders/types.php (hunk 1 above)
Comment (sjpadgett) [reply interface/orders/types.php]: I try to replace the xl with xl? whenever I can but missed this echo. Nice catch and thanks. Still working to improve the lookup and pickers so will leave open for awhile.

**Comment by sjpadgett** [comment]
Bringing in before gets to stale. Will add new PR for some additional updates when done.

**Pull request merged**